loopout.go

```diff
@@ -75,6 +75,24 @@ type executeConfig struct {
 	blockEpochChan  <-chan interface{}
 	timerFactory    func(d time.Duration) <-chan time.Time
 	loopOutMaxParts uint32
+	cancelSwap      func(context.Context, *outCancelDetails) error
+}
+
+func newExecuteConfig(sweeper *sweep.Sweeper, statusChan chan<- SwapInfo,
```

Comment:
If we don't have any logic in here, why are we using a function to initialize the struct? I find it much more readable to have a struct initialization block than a long list of function arguments.

Reply:
I somewhat prefer the constructor because we use `executeConfig` in so many places, albeit mostly tests, easy way to enforce that all fields are set. Not overly attached, reverted.